infra-templates/vxlan/9/README.md

```diff
@@ -0,0 +1,11 @@
+## VXLAN Networking
+
+Rancher networking plugin using VXLAN overlay.
+
+### Open Ports
+
+Traffic to and from hosts requires UDP port `4789` to be open.
+
+### Upgrade options
```

Comment:
Due to dependencies check in Cattle, deleting "vxlan" infra service is not possible without deleting all the existing stacks, scheduler and healthcheck. At this point it's almost similar to creating a new environment.